FirebaseAuth/Sources/Backend/FIRAuthBackend.h

```diff
@@ -423,16 +444,16 @@ typedef void (^FIRSignInWithGameCenterResponseCallback)(
  */
 @protocol FIRAuthBackendRPCIssuer <NSObject>
 
-/** @fn asyncPostToURLWithRequestConfiguration:URL:body:contentType:completionHandler:
-    @brief Asynchronously seXnds a POST request.
+/** @fn asyncCallToURLWithRequestConfiguration:URL:body:contentType:completionHandler:
+    @brief Asynchronously sends a POST request.
```

Comment:
nit: Please update this `@brief` and the `@param handler` documentation since this method is no longer exclusively for POST requests.